### PR TITLE
Removed white space in layout

### PR DESCRIPTION
**Pull request by suyesh**
Not sure what is the purpose of this white space but it bothers me every time.

### CODE REVIEW

**Comment by twitnithegirl** [comment]
@suyesh will you please attach a picture or description of whee this white space lives and what it looks like after changing it.

**Comment by suyesh** [comment]
**Before**
screen shot 2018-08-06 at 11 04 52 am

**After**
screen shot 2018-08-06 at 11 05 13 am

**Comment by twitnithegirl** [comment]
I thought you meant a whitespace in the actual UI. 😂😂😂😂😂😂😂😂😂😂😂😂
So much facepalm.
Never quit drinking coffee y'all!

File: railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt [hunk 1]
```diff
@@ -9,10 +9,10 @@
     <%%= stylesheet_link_tag    'application', media: 'all' %>
     <%- else -%>
       <%- unless options[:skip_turbolinks] -%>
-    <%%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (twitnithegirl) [inline railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt]: yep that's an extra space

**Review by twitnithegirl — APPROVED** [review]

**Comment by matthewd** [comment]
The purpose is visual alignment with the following line. If we're going to change it, there's another `stylesheet_link_tag` just above (which notably doesn't even have something to align with 🤔).

**Comment by suyesh** [comment]
@matthewd Makes sense. But i never noticed the alignment, The only thing i notice every time is the whitespace and manually remove it.

**Comment by twitnithegirl** [comment]
@matthewd @suyesh FWIW i don't really think the alignment is that important here since they aren't back to back. I could take it or leave it really. Is there a style guide for this?

**Comment by suyesh** [comment]
@twitnithegirl I am not sure of the style guide for this. I just find those spaces catching my eye all the time when i generate a new project. I usually move the javascript tag to the bottom above the </body> so the alignment doesn't mean anything for me. Even if i don't, it's just one of those nuance i notice all the time and fix it.

**Comment by rafaelfranca** [comment]
@dhh would have an opinion about those whitespaces.

**Comment by dhh** [comment]
Whitespace alignment is intentional. I'd like to keep these. I don't think there's any way we can codify these in a Rubocup rule.

**Comment by rafaelfranca** [comment]
Not for this case. What is important is the generated ERB file so a Rubocop rule will not be able to catch that, but we can have a test for it.

I'm closing this PR given the alignment is intentional. Thank you

**Pull request closed without merging**